test/update-version-readme-test.js

```diff
@@ -1,3 +1,13 @@
+/**
+ * update-version-readme-test.js
+ *
+ * A unit test for update-version.js
+ *
+ * Copies the fixtures to the temp folder and runs update-version
+ * After that the ( fixture ) files in the temp folder get compared with the expected folder
+ *
```

Comment:
Redundant newline.

Reply:
What do you mean, its the extra line under "After that the" unneccesery